### DISCUSSION TITLE
Youtube integration

### DISCUSSION THREAD

**Discussion by VogtAI** [edumeet/edumeet#830] [Ideas]
Enter a youtube link and everyone can see the video playing at the same time. The person that shared can pause and resume the video. It would also be cool if you could minimize this video in order to just listen to music from youtube. Additionally it would be great if it could automatically query for the title of youtube video, besides posting the url directly.

**Comment by misi** [comment]
Thanks! If you can please send Pull Request based on develop branch. I would happy to merge it.

**Comment by dadon** [comment]
Did you manage to do that?
I tried to implement something like this, but I faced a problem - if participants are watching a video without headphones, you can hear the echo of the video from other participants. 
The only way to solve this problem is to broadcast the video through mediasoup, in this case the browser will remove the echo.

**Comment by Aleeeeeeeex** [comment]
@dadon Cant moderator just mute all microphones on video play? Would you care to share the code for video playing? did you make a trigger function for moderator to start videos?
